vllm/worker/cpu_model_runner.py

```diff
@@ -216,11 +216,18 @@ def _prepare_prompt(
         slot_mapping = torch.tensor(slot_mapping,
                                     dtype=torch.long,
                                     device=self.device)  # type: ignore
+        max_seqlen = max(seq_lens)
+        tmp = [0]
+        tmp.extend(seq_lens)
+        seqlen = torch.tensor(tmp)
```

Comment:
nit: I think this looks a bit cleaner?
```suggestion
        seqlen = torch.tensor([0] + seq_lens)
```